README.md

```diff
@@ -1,7 +1,17 @@
-# OpenAPI Transformer Toolkig
+# OpenAPI Transformer Toolkit
 
 Effortlessly automate your design-first API development workflow by generating JSON schemas and TypeScript types from an OpenAPI specification.
 
+## Table of Contents
+
+- [Installation](#installation)
+- [CLI](#cli)
+- [Programmatic Usage](#programmatic-usage)
+  - [Generate JSON Schemas from OpenAPI](#generate-json-schemas-from-openapi)
+  - [Generate TypeScript Types from OpenAPI](#generate-typescript-types-from-openapi)
+  - [Generate TypeScript Types from JSON Schemas](#generate-typescript-types-from-json-schemas)
+- [Example](#example)
```

Comment:
can we use a tool to generate this dynamically please? I know for sure this will stop working otherwise as soon as we change something and we'll surely forget to update the ToC. There are tools to do this, but clearly we also need to integrate them in the workflow, e.g. a precommit hook that runs and commits the results.

Reply:
Sure, I will have a look at it 👍

Reply:
Updated ✅